.gitattributes

```diff
@@ -0,0 +1 @@
+secrets/* filter=git-crypt diff=git-crypt
```

Comment:
This needs to be secrets/**, otherwise it doesn't count subdirectories!

Reply:
Done.